types/FrameTypes.ts

```diff
@@ -29,31 +29,20 @@ export type FrameType = {
 
 export type Frame = TextFrame | ImageFrame;
 
-export type ImageUrlVariableSource = {
-    sourceType: ImageSourceTypeEnum.urlVariable;
+export type ImageVariableSource = {
+    sourceType: ImageSourceTypeEnum.variable;
     variableId: string;
 };
 export type ImageUrlSource = {
     sourceType: ImageSourceTypeEnum.url;
     url: string;
 };
-export type ImageConnectorVariableSource = {
-    assetId: string;
-    connectorId: string;
-    sourceType: ImageSourceTypeEnum.connectorVariable;
-    url: string;
-    variableId: string;
-};
 export type ImageConnectorSource = {
     assetId: string;
     connectorId: string;
     sourceType: ImageSourceTypeEnum.connector;
 };
-export type ImageFrameSource =
-    | ImageUrlVariableSource
-    | ImageUrlSource
-    | ImageConnectorVariableSource
-    | ImageConnectorSource;
+export type ImageFrameSource = ImageUrlSource | ImageConnectorSource;
```

Comment:
```suggestion
export type ImageFrameSource = ImageUrlSource | ImageConnectorSource | ImageVariableSource;
```
Not sure if that's to be in scope of this ticket but beside `ImageUrlSource`, removing the two other types does NOT make tests fail.

I'd say even the tiniest test to lock those expected types somewhere would be good to not accidentally remove them.

Reply:
my point was to remove the Variable type, since it's not being used by the FrameController now